src/styles/blocks/stars--1.css

```diff
@@ -0,0 +1,44 @@
+.stars--1 {
+  display: flex;
+  margin-top: 4px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). In this case, you are setting a top margin on '.stars--1'. If there's a need to add space around this element, consider using only bottom margin on the elements above or only top margin on the elements below to avoid potential margin collapse.